examples/next-prisma-starter/src/pages/_app.tsx

```diff
@@ -81,16 +82,33 @@ export default withTRPC<AppRouter>({
   /**
    * Set headers or status code when doing SSR
    */
-  responseMeta({ clientErrors }) {
-    if (clientErrors.length) {
-      // propagate http first error from API calls
+  responseMeta(opts) {
+    const ctx = opts.ctx as SSRContext;
+
+    if (ctx.redirectTo) {
+      // Do redirect if requested
       return {
-        status: clientErrors[0].data?.httpStatus ?? 500,
+        status: 303, // 303 See other
+        headers: {
+          location: ctx.redirectTo,
+        },
+      };
+    }
```

Comment:
Looks better, but I'm still not sure why this is needed/useful

Reply:
Removed for now.